src/installer/installer.go

```diff
@@ -231,6 +259,15 @@ func (i *installer) extractIgnitionToFS(ignitionPath string) (err error) {
 	return err
 }
 
+func (i *installer) generateSshKeyPair() error {
+	i.log.Info("Generating new SSH key pair")
+	if _, err := i.ops.ExecPrivilegeCommand(utils.NewLogWriter(i.log), "ssh-keygen", "-q", "-f", "/root/.ssh/id_rsa", "-N", ""); err != nil {
```

Comment:
Why use ExecPrivilegeCommand if `/root/ssh` is mounted?

Reply:
Because `ssh-keygen` doesn't exist on the container.